README.md

```diff
@@ -1,14 +1,103 @@
-# xCell
+![](https://raw.githubusercontent.com/JaimeRZP/Cosmoteka_tutorials/master/docs/src/assets/cosmoteka_logo.png)
 [![Coverage Status](https://coveralls.io/repos/github/xC-ell/xCell/badge.svg?branch=master)](https://coveralls.io/github/xC-ell/xCell?branch=master)
```

Comment:
I'd put the badges at the top, but not strong feelings.

Reply:
Done!